### PR TITLE
DEV-7243 update rubocop to use ruby 2.5.8 parser

### PR DESCRIPTION
**Pull request by ghouston**
ticket: https://proctorujira.atlassian.net/browse/DEV-7243

### CODE REVIEW

File: style/ruby/.rubocop.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # https://raw.githubusercontent.com/thoughtbot/hound/master/config/style_guides/ruby.yml
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.5.8
```
Comment (tmiller) [inline style/ruby/.rubocop.yml]: I believe you need to drop the `.8` here based on the documentation
https://docs.rubocop.org/rubocop/1.0/configuration.html#setting-the-target-ruby-version

Comment (ghouston) [reply style/ruby/.rubocop.yml]: I've run with 2.5.8 locally.  let me give 2.5 a try.

Comment (jamescook) [reply style/ruby/.rubocop.yml]: @ghouston Also from the docs it looks like Rubocop can simply use our `.ruby-version` file so we don't have to keep it in sync in two places:

> ... RuboCop will then check your project for .ruby-version and use the version specified by it.

Comment (ghouston) [reply style/ruby/.rubocop.yml]: 2.5 is working just as well.  i checked the warnings that 2.3 gave and they have gone away with 2.5.  update pushed.

**Review by tmiller — APPROVED** [review]

File: style/ruby/.rubocop.yml (hunk 1 above)
Comment (jamescook) [reply style/ruby/.rubocop.yml]: Nevermind what I said, I just realized this isn't the Archimedes repo 😅

**Review by jamescook — APPROVED** [review]

**Pull request merged**